lib/Client.js

```diff
@@ -95,12 +84,32 @@ function Client(config) {
 }
 inherits(Client, EventEmitter);
 
+Client.prototype.initHandle = function() {
```

Comment:
This should be underscored (`_initHandle`) since it's not part of the public API.